autoload/lsp/textedit.vim

```diff
@@ -60,7 +60,6 @@ def Set_lines(lines: list<string>, A: list<number>, B: list<number>,
 
   var new_lines_len: number = new_lines->len()
 
-  #echomsg $"i_0 = {i_0}, i_n = {i_n}, new_lines = {string(new_lines)}"
```

Comment:
I left these debug messages in the plugin so that these can be used later to debug issues with the Set_lines() function.
Instead of removing these debug messages, we can change the call to :echomsg to instead use Debug() function.

Reply:
this PR based on #262

Reply:
the record still in the git history if you like to bring it back in case to debug or something.
and yes, if you preferred, i think we can add a util.DebugMsg() later, for tidy, so i just removed them.

Reply:
ok, reverted it as you wish.